lib/super_good/solidus_taxjar/overrides/request_override.rb

```diff
@@ -0,0 +1,15 @@
+module SuperGood
```

Comment:
Maybe we namespace this under `Taxjar::Api` also?
I don't have a super strong opinion on that.